docs/modules/ROOT/pages/how-tos/redis/no_active_leader.adoc

```diff
@@ -0,0 +1,27 @@
+= Fix wrong/invalide leader after rollout/update
+
+IN certain situations it might happen, that the sentinels have the the master set to an ip address that doesn't exist anymore (if all pods have been restarted and the IPs changed). The most common thing that happens in this scenario is, that two of the pods are crashlooping and one pod is running as a `slave` and has the master set to an IP address that doesn't exist anymore.
```

Comment:
```suggestion
In certain situations it might happen, that the sentinels have the master set to an ip address that doesn't exist anymore (if all pods have been restarted and the IPs changed). The most common thing that happens in this scenario is, that two of the pods are crash looping and one pod is running as a `slave` and has the master set to an IP address that doesn't exist anymore.
```